plugins/line-numbers/prism-line-numbers.js

```diff
@@ -44,8 +44,8 @@
 
 			codeLines.forEach(function (line, lineNumber) {
 				lineNumberSizer.textContent = line || '\n';
-				var lineSize = lineNumberSizer.getBoundingClientRect().height;
-				lineNumbersWrapper.children[lineNumber].style.height = lineSize + 'px';
+				var lineSize = getComputedStyle(lineNumberSizer).lineHeight;
```

Comment:
Can we assume that `getComputedStyle(lineNumberSizer).lineHeight` is equivalent to `lineNumberSizer.getBoundingClientRect().height + 'px'`? Especially cross browser?

(Line Numbers is one of our most-used plugins, so I don't want to break websites just because this change seems sound.)

Reply:
How about we set lineSize to be `lineNumberSizer.getBoundingClientRect().height + 'px'` by default,only when `lineNumberSizer.getBoundingClientRect().height` equals 0 will we set it to `getComputedStyle(lineNumberSizer).lineHeight`.

Reply:
@RunDevelopment Request for your comments. :)

Reply:
(Thanks for the reminder)

I tested your solution (without fallback) and it doesn't work with soft wrap.
<details>

![Screenshot from 2020-05-11 15-04-12](https://user-images.githubusercontent.com/20878432/81564904-c05f3b00-9398-11ea-8632-920119291f50.png)

Should be:

![Screenshot from 2020-05-11 15-13-23](https://user-images.githubusercontent.com/20878432/81565673-010b8400-939a-11ea-8bd3-82a4a965974c.png)

</details>

I don't think there's a lot we can do to fix this. The browser needs to break and wrap the text for us.